Cargo.toml

```diff
@@ -65,12 +65,16 @@ targets = ["aarch64-apple-darwin", "aarch64-unknown-linux-gnu", "x86_64-apple-da
 publish-jobs = ["homebrew"]
 # Whether cargo-dist should create a Github Release or use an existing draft
 create-release = true
-# Publish jobs to run in CI
+# Which actions to run on pull requests
 pr-run-mode = "plan"
 # A prefix git tags must include for cargo-dist to care about them
 tag-namespace = "libsql-server"
 # Whether to install an updater program
 install-updater = false
+# additional setup steps
+github-build-setup = "../templates/libsql-server-release-build-setup.yml"
+# Path that installers should place binaries in
+install-path = "CARGO_HOME"
```

Comment:
why is that necessary?

Reply:
It seems like fresh `cargo dist` version add this by itself
I tried to remove this bit but `cargo dist init` returned it back